Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -2584,34 +2724,40 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "111":
-    id: "111"
-    taskid: f50bfd4c-7402-4bb8-8047-489f3d93d495
+  "124":
+    id: "124"
+    taskid: c44f7015-582f-461c-8a5d-52e5748adbfa
     type: regular
     task:
-      id: f50bfd4c-7402-4bb8-8047-489f3d93d495
+      id: c44f7015-582f-461c-8a5d-52e5748adbfa
       version: -1
-      name: Close alert (close now)
-      description: Close the current alert because auto-close was selected,
-      script: Builtin|||closeInvestigation
+      name: Set playbook stage grid field (triage)
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of command:
```

Comment:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Instead of a value, you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of the command:
```